### ISSUE TITLE
Thanks for this excellent firmware!

### ISSUE THREAD

**Issue by madcock** [eccherda/ch552g_mini_keyboard#3]
I created my own fork here:
https://github.com/madcock/ch552g_mini_keyboard

Sorry I didn't really split out the changes into pieces, but I included a few minor fixes that could be added back in to the main repo here. For instance, when going into bootloader mode it would change the colors when holding the encoder in while plugging in USB, but it didn't change the colors when pressing all buttons at once. Nothing major though, just a little cleanup.

I decided to try to get as many button mappings as possible, and ended up with a total of 36. That's 9 sets of 4 buttons (including encoder, but not using the rotation of it for mapping). To get enough space for that I removed the LOOP mode and also removed the AUTO configurations. The 36 buttons mapped well to the unused function keys from F13 to F24, and combos of those with CTRL and ALT. Any of the buttons should still work with your original configuration features (except auto), including mouse control, key and mouse sequences, etc.

TL;DR: I sacrificed some visual effect and some features to get as many button mappings as possible. :)

Thanks again for sharing this great work. Your guide made it easy to flash the initial firmware, and the firmware itself makes additional updates easy.
